generators/server/templates/pom.xml.ejs

```diff
@@ -77,7 +77,7 @@
         <maven.compiler.source>${java.version}</maven.compiler.source>
         <maven.compiler.target>${java.version}</maven.compiler.target>
         <start-class><%= packageName %>.<%= mainClass %></start-class>
-        <argLine>-Djava.security.egd=file:/dev/./urandom -Xmx256m</argLine>
+        <argLine>-Djava.security.egd=file:/dev/./urandom -Xmx1G</argLine>
```

Comment:
Not sure, this is to test.
The threshold was for mongo mostly (remember, we struggled on that). But changes introduced may have fixed this issue since then